lab_3_generate_by_ngrams/main.py

```diff
@@ -121,6 +180,18 @@ def decode(self, encoded_corpus: tuple[int, ...]) -> Optional[str]:
         In case of corrupt input arguments, None is returned.
         In case any of methods used return None, None is returned.
         """
+        if not isinstance(encoded_corpus, tuple):
+            return None
+
+        decoded_tokens = self._decode(encoded_corpus)
+        if not decoded_tokens:
+            return None
+
+        decoded_text = self._postprocess_decoded_text(decoded_tokens)
+        if not decoded_text:
```

Comment:
Instead of this - return `decoded_text or None`

Reply:
fixed, I guess?